src/Illuminate/Foundation/Console/stubs/test.stub

```diff
@@ -16,6 +16,7 @@ class DummyClass extends TestCase
      */
     public function testExample()
     {
+        $this->markTestIncomplete('Incomplete feature test at '.self::class);
         $this->assertTrue(true);
```

Comment:
I'm not sure that understand you properly, which exactly line throws an exception?